src/modules/currencyWallets/api.js

```diff
@@ -85,6 +89,9 @@ export function makeCurrencyApi (
   const engine = () => getCurrencyWalletEngine(ai.props.state, keyId)
   const plugin = () => getCurrencyWalletPlugin(ai.props.state, keyId)
 
+  const io = getIo(ai.props.state)
```

Comment:
This should just be `ai.props.io`. The whole `getIo` thing is being phased out.

Reply:
Actually, just `const shapeshiftApi = makeShapeshiftApi(ai.props.io)`. Keep it simple. :)